Jint/Jint.csproj

```diff
@@ -29,9 +29,27 @@
     <None Include="../README.md" Pack="true" PackagePath="\"/>
   </ItemGroup>
 
+  <!-- For checking generated output-->
+
+  <PropertyGroup>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>Generated</CompilerGeneratedFilesOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="$(CompilerGeneratedFilesOutputPath)/**/*.cs" />
+    <None Include="$(CompilerGeneratedFilesOutputPath)/**/*.cs" />
+    <Content Include="$(CompilerGeneratedFilesOutputPath)/**/*.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Jint.SourceGenerators\Jint.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Esprima" />
     <PackageReference Include="Meziantou.Analyzer" PrivateAssets="all" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
Hey let the man rebase in peace 😆

I've been experimenting, but [the new switch-based compiler feature for string matching](https://github.com/dotnet/roslyn/pull/66081) probably will make need obsolete. Jint already caches hash code for `Key` as we need to traverse different contexts and hash would be otherwise needed for each context level lookup.